cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -54,23 +46,19 @@ OPTIONS
         Provides the path to be taken as the root of the source.
         Source will be looked up relative to the given source root
         path.
-
-    -c <config-file>
-    --config <config-file>
-        Provides the path to a Ballerina configuration file.
-        The configuration file format is a subset of the TOML file
-        format. It supports key/value pairs, tables, comments, and
-        string, int, float and boolean value types. If a config
-        file is not specified using [-c|--config], Ballerina looks
-        for a default configuration file named `ballerina.conf`
-        in the directory the source file(s) reside in.
-
-    -B <key=value>
-        Provides Ballerina VM options to be used by the runtime
-        when executing a Ballerina program or module. VM options
-        are key/value pairs. If multiple VM options need to be
-        provided, each option has to be specified using its
-        own -B flag.
+        
+    configs(--key=value)...
+        Sets Ballerina environment parameters as key/value pairs.
+        If multiple parameters need to be provided, each parameter
+        has to be preceded by '--'. Details of the
+        environment parameters that the Ballerina standard library
+        modules support can be found in the relevant API
+        documentation.
+	        
+    --
+        The optional argument ‘--’ terminates all configs. Any succeeding 
+        arguments are treated as args of the program even if they start with '--'. 
+        This is required only if a specific arg needs to start with '--'.
```

Comment:
If I understand correctly, what this means is the `--` is required if the user wants to give an argument starting with `--` to his program right?

Reply:
Yes. I change the description as suggested and seems fine.